src/NexusMods.DataModel/Settings/DataStoreSettingsBackendEntity.cs

```diff
@@ -0,0 +1,12 @@
+using NexusMods.Abstractions.Serialization.Attributes;
+using NexusMods.Abstractions.Serialization.DataModel;
+
+namespace NexusMods.DataModel.Settings;
+
+[JsonName("NexusMods.DataModel.Settings.DataStoreSettingsBackendEntity")]
+internal record DataStoreSettingsBackendEntity : Entity
```

Comment:
This probably should be a `record struct` to avoid boxing, since the internal type is already on heap.

Reply:
This can't be a struct.

Reply:
Ah, that's fine. Sleepy me didn't notice.